api/src/constants.js

```diff
@@ -7,3 +7,20 @@ export const RESOURCE_VERBS = {
   UPDATE: 'UPDATE',
   CREATE: 'CREATE',
 };
+
+export const FILE_PATH = 'app-web/topicRegistry/';
+
+export const COMMIT_MESSAGE = {
+  CREATE: 'add new topic',
+  UPDATE: 'edit topic',
+};
+
+export const PR_TITLE = {
+  CREATE: 'Suggest new topic to the devhub',
+  UPDATE: 'Suggest an edit to topic',
+};
+
+export const PR_BODY = {
+  CREATE: 'Suggest a new topic',
```

Comment:
we talked about this

Reply:
The topic description is not a constant, so does not lie in the constants file. It is dynamically loaded, when the pull request is created. I have included it in the createPullRequest function